src/main/java/de/techdev/portal/domain/user/update/TokenDeleter.java

```diff
@@ -1,17 +1,19 @@
 package de.techdev.portal.domain.user.update;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.oauth2.common.OAuth2AccessToken;
 import org.springframework.security.oauth2.provider.token.TokenStore;
 import org.springframework.stereotype.Service;
 
 import java.util.Collection;
 
-import static de.techdev.portal.core.OauthAuthorizationServerConfiguration.TRACKR_PAGE_CLIENT;
-
 @Service
 class TokenDeleter {
 
+    @Value("${trackr.oauth.webClientId}")
+    private String trackrWebpageClientId = "trackr-page";
```

Comment:
See comment on line 15. Do not initialize this, since the value is/should be set by @Value.